capabilities/code-maintainability.md

```diff
@@ -37,77 +56,91 @@ Don't worry if the description doesn't exactly match your situation. The descrip
 
 ### Previously Written Code
 
-1. **Brittle Codebase:** Changing any code is time-consuming, complex, and error prone.
-2. **Fairly Complex:** Most changes require significant refactoring, and it's difficult to predict the impact of changes on the overall system.
-3. **Partially Modular:** Most parts of the system are modular and easy to update, but some are complex and difficult to work with.
-4. **Well Organized:** When changes are made to the existing codebase, they don’t tend to require much rework.
+1. **Brittle Codebase:** Changing any code is time-consuming, complex, and prone to error.
+2. **Fairly Complex Codebase:** Most changes require significant refactoring, and it's difficult to predict the impact of changes on the overall system.
+3. **Partially Modular Codebase:** Most parts of the system are modular and easy to update, but some are complex and difficult to work with.
+4. **Well-organized Codebase:** When changes are made to the existing codebase, they don’t tend to require much rework.
+
+Now average your scores for the two responses and that will be your overall score for this capability.
+
+<ed> By "two responses" do we mean New Code and Previously Written Code? I think we could make this instruction clearer for the reader. My understanding is that the reader is to score themselves on each of the 4 areas in each of the two "subcategories" above (New Code and Previously Written Code). Do they then average the scores of each subcategory or do they average the scores of ALL areas (which would be 8 areas total)? </ed> 
```

Comment:
So most of the capabilities have a simple 1-4 rating. However, some capabilities (like this one), are broken into multiple areas of focus, which need to be rated 1-4 individually. Then you can average those areas of focus together to get a single rating for the capability. You can see what we did in our sample doc here: https://docs.google.com/spreadsheets/d/1KNjRLxNfM6Jt_eaLx_B1NuYsiEtKAqzTJ9AbCYm1b10/edit

Code Maintainability, Continuous Delivery, and Test Data Management are the only capabilities that had to be broken into multiple areas of focus.

Reply:
I wasn't sure where you were going with that. It seems covered in the paragraph above. So I just removed it and it reads fine to me now. Is there something I'm missing?

Reply:
I'm having trouble translating the scoring that you're using for the capability to the 8 descriptions that follow. 

For example, take "Brittle codebase." The reader would score a 1 if...brittle codebase is completely missing from their team or organization? A 2 if there is a lot of room for improvement around brittle codebase, a 3 if there is some room for improvement, and a 4 if your team or organization is exemplary in the area of...brittle codebase? It doesn't match up for me, at least the way the scores are worded. 

To me, it seems more logical that the reader would rank each scenario based on how accurately it describes their current situation. 1 for does not occur, 2 for sometimes occurs, 3 for often occurs, and 4 for always occurs. 

But, you've been using this assessment with clients, so perhaps I'm overthinking this.

Reply:
Ok, let's keep it how it is for now, merge it, and then we can revisit the topic over our next video chat and make subsequent updates if needed.